src/views/examples/Register.js

```diff
@@ -33,126 +35,114 @@ import {
   Col,
 } from "reactstrap";
 
-const Register = () => {
-  return (
+
+
+
+function checkpassword(x,y) {
+  if(!x || !y) {
+    document.getElementById("Passwordmatch").style.color="red"
+    document.getElementById("Passwordmatch").innerHTML="Please set up your Password";
+  }
+  else if(x!==y) {
+    document.getElementById("Passwordmatch").style.color="red";
+    document.getElementById("Passwordmatch").innerHTML="Passwords did not match!";
+    document.getElementById("submit").disabled=true;
+  }
+  else if(x===y){
+    document.getElementById("submit").disabled=false;
+    document.getElementById("Passwordmatch").innerHTML= "";
+
+  }
+}
+
+function handlesubmit() {
+  //api
+  
+}
```

Comment:
1. Don't define a function like this. Be it inside functional component like const functionname = () => {}.
2. Disable Register button till all fields are filled without failing any validation.
3. Kindly check the password in other ways as well like if some user make changes in password after confirm password then also it is giving same error.